src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Configuration;
+
+namespace APIViewWeb.Pages.Assemblies
+{
+    public class IndexModel : PageModel
+    {
+        public IndexModel(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public Response<BlobsFlatSegment> Segment { get; set; }
+
+        public async Task OnGetAsync()
+        {
+            string connectionString = Configuration.GetValue<String>("APIVIEW_STORAGE");
+            var service = new BlobServiceClient(connectionString);
+            var container = service.GetBlobContainerClient("hello");
```

Comment:
It'd probably be a good idea to add another `APIVIEW_STORAGE_CONTAINER` environment variable so you can swap this out as needed.